File: TPTD3/Stocker.java
public class Stocker {
    private int reference;
    private int code;
    private int quantite;
    public Stocker(int reference, int code, int quantite) {
        this.reference = reference;
        this.code = code;
        this.quantite = quantite;
    }
    public int getReference() {
        return reference;
    }
    public void setReference(int reference) {
        this.reference = reference;
    }
    public int getCode() {
        return code;
    }
    public void setCode(int code) {
        this.code = code;
    }
    public int getQuantite() {
        return quantite;
    }
    public void setQuantite(int quantite) {
        this.quantite = quantite;
    }
    @Override
    public String toString() {
        return "Stocker [reference=" + reference + ", code=" + code + ", quantite=" + quantite + "]";
    }
}

File: TPTD4/Stocker.py
class Stocker:

    def __init__(self,reference,code,qte) -> None:
        self._reference = reference
        self._code = code
        self._qte = qte

    def __repr__(self) -> str:
        return "Article %d dans l'entrepot %s est disponible en qte %d"+"\n"

    @property
    def getRef(self):
        return self._reference
    
    @property
    def getCode(self):
        return self._code
    
    @property
    def getQte(self):
        return self._qte
    
    def setRef(self,ref):
        self._reference = ref
    
    def setCode(self,l):
        self._code = l
    
    def setQte(self,p):
        self._qte = p